src/public/tier1/utltshash.h

```diff
@@ -617,7 +617,6 @@ inline UtlTSHashHandle_t CUtlTSHash<T,BUCKET_COUNT,KEYTYPE,HashFuncs,nAlignment>
 {
 	Assert( pElement );
 	HashFixedData_t *pFixedData = (HashFixedData_t*)( (uint8*)pElement - offsetof( HashFixedData_t, m_Data ) );
-	Assert( m_EntryMemory.IsAllocationWithinPool( pFixedData ) );
```

Comment:
Why is this removed?

Reply:
The function `IsAllocationWithinPool` doesn't exist.

Reply:
Why can I compile it now?

Reply:
Probably something about how MSVC handles asserts, I know I had to remove this to get GCC or clang to compile it.

Reply:
Can you comment it as a TODO? It's present in later version of mempool.